lib/vcr/configuration.rb

```diff
@@ -496,7 +496,8 @@ def initialize
         :match_requests_on => RequestMatcherRegistry::DEFAULT_MATCHERS,
         :allow_unused_http_interactions => true,
         :serialize_with    => :yaml,
-        :persist_with      => :file_system
+        :persist_with      => :file_system,
+        :persister_options => {}
```

Comment:
What is the reasoning for introducing a full set of options here? I would have expected something more along the lines of a single option ```:downcase_cassette_names => false```.

Reply:
I was following a pattern here that I've seen often in Rails, where a given configuration option will have its own option hash if it's destined to be used by that other piece of config. For example, you can set `config.session_store` to `:memory`, `:redis`, etc, and whichever adapter you specify there can be further customized by what you pass to `config.session_options`.

I do agree it might be a bit overkill since there's only one supported option right now, so I'm happy to change this if others feel the same.